Bucketeer2/Sources/Internal/Database/Migration/Migration.swift

```diff
@@ -0,0 +1,20 @@
+protocol Migration {
+    func migration() throws
+}
+
+final class Migration1to2: Migration {
+    private let db: SQLite
+
+    init(db: SQLite) {
+        self.db = db
+    }
+
+    func migration() throws {
+        let evaluationTable = SQLite.Table(entity: EvaluationEntity())
```

Comment:
Don't we need to drop the table first before creating it?

Reply:
Added SQL for deletion, as you pointed out. Thanks! https://github.com/ca-dp/bucketeer-ios-sdk/pull/83/commits/5ccafa24e9aa5cd8b95bfd731fe7fa7495adb068

Reply:
Thank you!

Would it be possible for you to add a unit test for the migration process?

E.g.
https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/database/migration/MigrationTest.kt

Reply:
Sorry, I forgot!
I'll add it.

Reply:
Added the test! ✅   https://github.com/ca-dp/bucketeer-ios-sdk/pull/83/commits/73bf5b1a2d1b9f17ed84defd3178d9cedab18d88